wiki/en/Installation-for-Linux.md

```diff
@@ -54,6 +54,10 @@ For installers on other distributions, see their package managers and [Repology]
 
 ## Set up your hardware
 
+If your distribution uses JACK as sound server, go to "Configure JACK with QjackCtl".
+
+If your distribution uses PipeWire by default, go to "Configure PipeWire".
```

Comment:
Same here: 

check out the section on `[configuring PipeWire](Installation-for-Linux#configure-pipewire)`